docs/content/ingestion/index.md

```diff
@@ -71,6 +72,7 @@ An example dataSchema is shown below:
 | dataSource | String | The name of the ingested datasource. Datasources can be thought of as tables. | yes |
 | parser | JSON Object | Specifies how ingested data can be parsed. | yes |
 | metricsSpec | JSON Object array | A list of [aggregators](../querying/aggregations.html). | yes |
+| rollup | Boolean | rollup or not | no (default == true) |
```

Comment:
agree with @fjy,   I am in favor of keeping rollup on by default at least for 0.9